### ISSUE TITLE
Reset orient tag for TGA, etc (EXIF)

### ISSUE THREAD

**Issue by Bxaa** [microsoft/DirectXTex#248]
Hi, walbourn
Is it possible to add arg like -auto_orient to fix wrong image orient
Yes, it can be done via -flip or else, but some image maybe correct orient (massive bach conversion)
Need to reset it in EXIF and auto flip image if orient set to 'flip'
(Actual for old targa formats)

Wrong tag
https://drive.google.com/file/d/1qBLxZdBfiq2RFV3ayfuuKc8r5Csd0yXd/view?usp=sharing

**Comment by walbourn** [comment]
Thanks for the clarification.

First, the TGA codec used by DirectXTex is 'hand-rolled'. It 's not part of WIC, so it doesn't have any formal EXIF or other metadata support beyond what I've implemented.

As such, I don't know what field of metadata in the file format is being interpreted as **Orientation** by the tool you are using. My guess is that it's ``TGA_HEADER.bDescriptor`` where the 4th and 5th bit indicate inverting X & Y. My reader respects these bits, but my writer always uses ``INVERTY``.

In other words: Just passing the TGA through **texconv** will already "reorient" the TGA to always use INVERTY.
